logs.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2016 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mixer
+
+import (
+	"encoding/json"
+	"io"
+	"time"
+)
+
+type (
+	// LogEntry is the top-level struct for logs data that will be
+	// exported by logging adapters.
+	LogEntry struct {
```

Comment:
I put in the mixer package thinking that there may be components that would want to use it beyond adapters (or out of the context of adapters) -- perhaps the some of the dispatch pieces or whatever.

If we don't think it will serve a purpose beyond the Adapters, I'm fine moving this all to adapters/logger.go.